### ISSUE TITLE
LTS - System Test fails with possible data corruption in readTxnWriteAutoScaleWithFailoverTest

### ISSUE THREAD

**Issue by sachin-j-joshi** [pravega/pravega#6644]
**Describe the bug**
It is observed that `readTxnWriteAutoScaleWithFailoverTest` fails consistently after #6541 and #6588.
Until we figure out the root cause we need to revert these two commits.

**To Reproduce**
Run system tests multiple times

**Screenshots**
None

**Additional information**
None

**Comment by RaulGracia** [comment]
This is duplicate of #6614.
